http/vertx-web-client/src/test/java/io/quarkus/qe/vertx/webclient/ChuckNorrisResourceIT.java

```diff
@@ -0,0 +1,189 @@
+package io.quarkus.qe.vertx.webclient;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.Every.everyItem;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
+
+import java.io.File;
+import java.net.HttpURLConnection;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.vertx.webclient.resources.JaegerTestResource;
+import io.quarkus.qe.vertx.webclient.resources.WireMockChuckNorrisResource;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.scenarios.annotations.EnabledOnNative;
+import io.restassured.response.Response;
+
+@QuarkusTest
+@QuarkusTestResource(WireMockChuckNorrisResource.class)
+@QuarkusTestResource(JaegerTestResource.class)
+public class ChuckNorrisResourceIT {
```

Comment:
This test is not using the test framework, please update it.

Reply:
Test updated